src/Packaging/Packaging.csproj

```diff
@@ -0,0 +1,165 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" InitialTargets="DownloadScannerCli">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(ScannerNetFxVersion);$(ScannerNetCoreAppVersion);$(ScannerNetCoreGlobalToolVersion)</TargetFrameworks>
```

Comment:
It is ok to move the constants here or to hardcode if we have only one project

Reply:
We use them in multiple places (for `ScannerNetCoreGlobalToolVersion` it is 2 places).